containers/opcua-device-gateway/entrypoint.sh

```diff
@@ -3,10 +3,10 @@ set -e
 
 # Try loading any values, which aren't already set, using tedge cli
 if command -V tedge >/dev/null 2>&1; then
-    if [ -z "$DEVICE_ID" ]; then
-        DEVICE_ID="$(tedge config get device.id 2>/dev/null ||:)"
-        echo "Using value from tedge: DEVICE_ID=$DEVICE_ID" >&2
-        export DEVICE_ID
+    if [ -z "$GATEWAY_THINEDGE_DEVICEID" ]; then
```

Comment:
for consistency it might be worth renaming the variable to:

```
GATEWAY_TEDGE_DEVICE_ID
```

Reply:
Yes basically all variables address now purely opc-gateway config